src/dfmt/indentation.d

```diff
@@ -197,7 +197,8 @@ private:
                 immutable currentIsNonWrapTemp = !isWrapIndent(arr[i])
                     && isTempIndent(arr[i]) && arr[i] != tok!")" && arr[i] != tok!"!";
                 if (arr[i] == tok!"static" && (arr[i + 1] == tok!"if"
-                        || arr[i + 1] == tok!"else") && (i + 2 >= index || arr[i + 2] != tok!"{"))
+                        || arr[i + 1] == tok!"else" || arr[i + 1] == tok!"foreach"
+                        || arr[i + 1] == tok!"foreach_reverse") && (i + 2 >= index || arr[i + 2] != tok!"{"))
```

Comment:
I would `import std.algorithm : among` and make this
```d
arr[i + 1].among!(tok!"if", tok!"else", tok!"foreach", tok!"foreach_reverse")
```
because it has a lot of cases now